test/apiMethods.test.ts

```diff
@@ -0,0 +1,371 @@
+import * as nf from "node-fetch";
+import { createReadStream } from "fs";
+import path from "path";
+import { jest } from "@jest/globals";
+import bitdotio from "../lib";
+import { ApiError } from "../lib/errors";
+
+function mockResponse(status: number, data: Record<string, any>): nf.Response {
+  return new nf.Response(JSON.stringify(data), {
+    status,
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+}
+
+describe("listDatabases", () => {
+  const b = bitdotio("v2_testtoken");
+
+  test("listDatabases ok", async () => {
+    const expected = ["foo", "bar"];
+    jest
+      .spyOn(nf, "default")
+      .mockResolvedValueOnce(mockResponse(200, { databases: expected }));
+    const result = await b.listDatabases();
+    expect(result).toEqual(expected);
+  });
+  test("listDatabases error", async () => {
+    const status = 400;
+    const data = { error: "whoops" };
+    jest.spyOn(nf, "default").mockResolvedValueOnce(mockResponse(status, data));
+    try {
```

Comment:
All of these async thrown error tests look odd to me. I think the idiomatic approach is using rejects: https://jestjs.io/docs/expect#rejects

something like 
```
test('rejects to octopus', () => {
  // make sure to add a return statement
  return expect(Promise.reject({name: 'octopus', status: 401})).rejects.toEqual(
    expect.objectContaining({name: 'octopus', status:401}) );
});
```

Reply:
You're right. I didn't realize I could do:
```js
await expect(b.someMethod()).rejects.toThrow(new ApiError(...));
```
will fix 🤦‍♀️